app/src/helpers/helpers.test.ts

```diff
@@ -42,3 +42,29 @@ test('getCounterValue should return a string for small counter numbers in the ti
 test('getCounterValue should return a string for large counter numbers in the title', () => {
   expect(getCounterValue(largeCounterTitle)).toEqual('8,756');
 });
+
+const shortArg = '--t';
+const longArg = '--test';
+const extraLongArg = '--test-run';
+const invalidShortArg = '--t';
+const invalidLongArg = '-test';
+
+test('isArgFormatValid should return true for short arguments', () => {
+  expect(isArgFormatValid(shortArg)).toEqual(true);
+});
+
+test('isArgFormatValid should return true for long arguments', () => {
+  expect(isArgFormatValid(longArg)).toEqual(true);
+});
+
+test('isArgFormatValid should return true for extra long arguments', () => {
+  expect(isArgFormatValid(extraLongArg)).toEqual(true);
+});
+
+test('isArgFormatValid should return false for improperly formatted short arguments', () => {
+  expect(isArgFormatValid(invalidShortArg)).toEqual(false);
+});
+
+test('isArgFormatValid should return false for improperly formatted long arguments', () => {
+  expect(isArgFormatValid(invalidLongArg)).toEqual(false);
+});
```

Comment:
Not your fault, but the initial test suite code lacked sub-suite structure. Test suites are typically structured into sub-suites, one per tested function, which provides a range of benefits:

1. Clear visual/mental separation between sub-suites
1. Proper scoping & isolation of sub-suite constants
2. Easiness to run only one sub-suite (with a little `.only` at sub-suite level)

`helpers.test.ts` tests three function: `linkIsInternal`, `getCounterValue`, and now `isArgFormatValid`. Can you separate group of tests for each into its own sub-suite by wrapping groups of tests into a `describe` call? See `inferUserAgent.test.ts` for an example.